tasks/scripts/k8s-topgun

```diff
@@ -24,4 +24,4 @@ go mod download
 
 go install github.com/onsi/ginkgo/ginkgo
 
-ginkgo -nodes=4 -race -keepGoing -slowSpecThreshold=300 -skip="$SKIP" ./topgun/k8s/ "$@"
+ginkgo -nodes=8 -race -keepGoing -slowSpecThreshold=900 -flakeAttempts=3 -flake-skip="$SKIP" ./topgun/k8s/ "$@"
```

Comment:
with the more conservative use of `resources.requests` (see the PR that makes this one possible), we can bump the number of concurrent nodes here as we're putting less work per node.

Reply:
ps.: `flakeAttempts`: https://github.com/onsi/ginkgo/pull/261